scripts/check-critical-libraries.sh

```diff
@@ -31,7 +31,7 @@ for REPO in ${REPOS_TO_CHECK[@]}; do
     TAG=$(getLatestReleaseTagForRepo $REPO)
     git clone $REPO -c advice.detachedHead=false --depth 1 --branch $TAG $TMP_DIR
     
-    nargo test -q --program-dir $TMP_DIR
+    nargo test --program-dir $TMP_DIR --format json
```

Comment:
```suggestion
    nargo test -q --program-dir $TMP_DIR
```

Reverting this change as we don't need the full output in this script.